Sources/CorePayments/Networking/HTTP.swift

```diff
@@ -10,7 +10,7 @@ class HTTP {
 
     init(
```

Comment:
Take it or leave it comment to add a docstring solely for the purpose of clarifying what `4 * 1024 * 1024` means

```suggestion
    /// Initializer
    /// - Parameters:
    ///   - urlSession: Apple's default instance of `URLSession.shared`.
    ///   - urlCache: Has a maximum memory usage of 4 megabytes (4 * 1024 * 1024 bytes). Does not cache to disk.
    ///   - coreConfig: The merchant's ``CoreConfig``.
    init(
```